### PR TITLE
Feature/sprind 13

### PR DESCRIPTION
**Pull request by sanderPostma**
There are TODO's like being able to turn off R|S signature format for RSA support and README documentation.
But it's usable in both Android & iOS so I better not block progress on other tasks.

### CODE REVIEW

File: packages/musap-react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# react-native-musap-rn
+
+## Getting started
+
+`$ npm install react-native-musap-rn --save`
```
Comment (nklomp) [inline packages/musap-react-native/README.md]: Whats up with the `react-native-xxx-rn`? That is 2 times RN

File: package.json [hunk 2]
```diff
@@ -1,49 +1,45 @@
 {
-  "name": "musap-react-native",
-  "main": "expo-router/entry",
+  "name": "@sphereon/musap-react-native-parent",
   "version": "1.0.0",
```
Comment (nklomp) [inline package.json]: 0.1.0

File: package.json [hunk 3]
```diff
@@ -1,49 +1,45 @@
...
-    "reset-project": "node ./scripts/reset-project.js",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
-    "web": "expo start --web",
-    "test": "jest --watchAll",
-    "lint": "expo lint"
+  "main": "index.js",
+  "repository": "https://github.com/Sphereon-Opensource/musap-react-native",
+  "author": "sanderPostma <spostma@sphereon.com>",
+  "license": "MIT",
```
Comment (nklomp) [inline package.json]: Improper license and author

File: packages/musap-react-native/android/build.gradle [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+
+afterEvaluate {
+    publishing {
+        publications {
+            aar(MavenPublication) {
+                groupId 'com.sphereon.musap'
+                artifactId 'musap-react-native'
+                version '1.0.0'
```
Comment (nklomp) [inline packages/musap-react-native/android/build.gradle]: Why 1.0.0?

File: packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/MusapModule.kt [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+            promise.reject("SIGN_ERROR", "Error preparing signature request: ${e.message}", e)
+        }
+    }
+
+    private fun attachSignature(orig: JWSObject, sig: MusapSignature): JWSObject {
+        try {
+            val header = orig.header.toBase64URL()
+            val payload = orig.payload.toBase64URL()
+            val signature = Base64URL.encode(transcodeSignature(sig.rawSignature))
+            return JWSObject(header, payload, signature)
```
Comment (nklomp) [inline packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/MusapModule.kt]: Where is this used? As we need Cbor/Cose as well, I hope we are not really depending on this

File: packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt [hunk 6]
```diff
@@ -0,0 +1,6 @@
+package com.sphereon.musap.models
+
+enum class SscdType(val value: String) {
+    TEE("TEE"),
```
Comment (nklomp) [inline packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt:4]: Where is eSim/eUICC and HSM, we would support for the project?

Please create a new ticket for that

File: packages/musap-react-native/package.json [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  "author": {
+    "name": "Sphereon",
+    "email": "dev@sphereon.com"
+  },
+  "license": "Apache-2.0",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "@sphereon/musap-native": "workspace:*",
+    "@sphereon/musap-react-native": "0.0.1-unstable.89+5df316cc",
```
Comment (nklomp) [inline packages/musap-react-native/package.json]: Why isn't this workspace:*?

**Review by nklomp — CHANGES_REQUESTED** [review]

File: packages/musap-react-native/README.md (hunk 1 above)
Comment (sanderPostma) [reply packages/musap-react-native/README.md]: Outdated, that was still from the template generator

File: package.json (hunk 2 above)
Comment (sanderPostma) [reply package.json]: Fixed
For the packages lerna is managing the versions which is at -0.0.1-unstable.134 now.

File: package.json (hunk 3 above)
Comment (sanderPostma) [reply package.json]: Fixed

File: packages/musap-react-native/android/build.gradle (hunk 4 above)
Comment (sanderPostma) [reply packages/musap-react-native/android/build.gradle]: Must have overlooked 
fixed in next commit

File: packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/MusapModule.kt (hunk 5 above)
Comment (sanderPostma) [reply packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/MusapModule.kt]: It's not, it in a orphan function attachSignature. I ripped JWS support out of the native code

Comment (sanderPostma) [reply packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/MusapModule.kt]: I cleaned up the orphaned methods

File: packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt (hunk 6 above)
Comment (sanderPostma) [reply packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt:4]: TEE is a common name for Secure Enclave and Android Keystore so you won't need different typescript code for iOS & Android.
For the rest I see only Yubikey support in their library and some vague SSCDs
https://github.com/methics/musap-android/tree/main/app/src/main/java/fi/methics/musap/sdk/sscd
https://github.com/methics/musap-ios/tree/main/Sources/sscd
Maybe eSim/eUICC is supported through "external"

Comment (sanderPostma) [reply packages/musap-react-native/android/src/main/kotlin/com/sphereon/musap/models/SscdType.kt:4]: https://sphereon.atlassian.net/browse/SPRIND-30 BTW

File: packages/musap-react-native/package.json (hunk 7 above)
Comment (sanderPostma) [reply packages/musap-react-native/package.json]: No that is its own package, removed. (This also explains why I had to set an override in SSI-SDK-crypto-extensions)

**Review by nklomp — APPROVED** [review]

**Pull request merged**